### PR TITLE
Use two different models for costing params

### PR DESCRIPTION
**Pull request by 0xOmarA**

### CODE REVIEW

File: radix-transactions/src/model/executable.rs [hunk 1]
```diff
@@ -62,14 +62,14 @@ impl From<(BlueprintId, GlobalAddress)> for PreAllocatedAddress {
 }
 
 #[derive(Debug, Clone, PartialEq, Eq, ScryptoSbor, ManifestSbor)]
-pub struct TransactionCostingParameters {
+pub struct TransactionCostingParametersV2 {
```
Comment (jakrawcz-rdx) [inline radix-transactions/src/model/executable.rs:65]: These structs do not use our conventional "versioning infra" (the macros, the wrappers, the traits, etc), so maybe we shouldn't use the `...V2` naming here.
Maybe something like "input costing params" vs "output costing params", or "receipt costing params"?
(unless this was already discussed and you are sure we want V1/V2)

File: radix-transactions/src/model/executable.rs [hunk 2]
```diff
@@ -79,6 +79,31 @@ impl Default for TransactionCostingParameters {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, ScryptoSbor, ManifestSbor)]
+pub struct TransactionCostingParametersV1 {
```
Comment (jakrawcz-rdx) [inline radix-transactions/src/model/executable.rs:83]: Can we add a note (regular comment) explaining why it is introduced?

(also, maybe in the PR description, some link to the investigation)

**Review by jakrawcz-rdx — COMMENTED** [review]
This should fix the issue indeed 👍

**Review by iamyulong — APPROVED** [review]

File: radix-transactions/src/model/executable.rs (hunk 1 above)
Comment (0xOmarA) [reply radix-transactions/src/model/executable.rs:65]: Sounds good to me, will switch the naming to indicate that our versioning infra is not used here.

**Pull request merged**